packages/bitcoin/src/bitcoin.utils.ts

```diff
@@ -277,6 +279,22 @@ export function getTaprootAddress({ index, keychain, network }: GetTaprootAddres
   return payment.address;
 }
 
+export function getNativeSegwitAddress({ index, keychain, network }: GetAddressArgs) {
+  if (!keychain) throw new Error('Expected keychain to be provided');
+
+  if (keychain.depth !== DerivationPathDepth.Account)
+    throw new Error('Expects keychain to be on the account index');
+
+  const addressIndex = deriveAddressIndexKeychainFromAccount(keychain)(index);
+
+  if (!addressIndex.publicKey) throw new Error('Expected publicKey to be defined');
+
+  const payment = getNativeSegWitPaymentFromAddressIndex(addressIndex, network);
```

Comment:
Did you mean to do `SegWit` in name here, just flagging bc we don't typ ...thou maybe more common to?

Reply:
That's a mistake, should stick to Segwit as one word, will change